spec/requests/api/v1/topics/retrieve_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'GET api/v1/topics', type: :request do
+  let(:password) { 'password' }
+  let(:user) { create(:user, password: password) }
+
+  context 'when is logged in' do
+    before do
+      params = {
+        user:
+          {
+            email: user.email,
+            password: password
+          }
+      }
+      post new_user_session_path, params: params, as: :json
+    end
+
+    it 'does return a successful response' do
+      get '/api/v1/topics', headers: auth_headers, as: :json
+      expect(response).to have_http_status(:success)
+    end
```

Comment:
You should test that the topics are actually fetched. Try adding some like 
````
let(:topic1) { create :topic }
let(:topic2) { create :topic }
````
Then you can check if the ids or labels matches.